README.md

```diff
@@ -1,2 +1,50 @@
 # UTBotJava-action
-GitHub action that runs UTBotJava for your project
+
+An action for generating unit tests and creating SARIF reports based on [UTBotJava](https://github.com/UnitTestBot/UTBotJava).
```

Comment:
based on [UTBotJava](https://github.com/UnitTestBot/UTBotJava) -> with the [UTBotJava](https://github.com/UnitTestBot/UTBotJava) engine